app/src/main/java/com/framgia/wsm/screen/requestoff/RequestOffViewModel.java

```diff
@@ -31,4 +51,110 @@ public void onStart() {
+
+    public void onPickTypeItemOffHaveSalary(View view) {
+        mDialogManager.dialogListSingleChoice(mContext.getString(R.string.off_have_salary_type),
+                R.array.off_have_salary_type, mCurrentPositionOffHaveSalaryType,
+                new MaterialDialog.ListCallbackSingleChoice() {
+                    @Override
+                    public boolean onSelection(MaterialDialog materialDialog, View view,
+                            int positionType, CharSequence charSequence) {
+                        setCurrentPositionOffHaveSalaryType(positionType);
+                        setCurrentOffHaveSalaryType(String.valueOf(charSequence));
+                        setLayoutOffHaveSalaryType(positionType);
+                        return true;
+                    }
+                });
+    }
+
+    private void setLayoutOffType(int positionType) {
+        if (positionType == PositionOffType.OFF_HAVE_SALARY) {
+            mSalaryFlag = false;
+            notifyPropertyChanged(BR.salaryFlag);
+        }
+        if (positionType == PositionOffType.OFF_NO_SALARY) {
+            mSalaryFlag = true;
+            notifyPropertyChanged(BR.salaryFlag);
+        }
+    }
+
+    private void setLayoutOffHaveSalaryType(int positionType) {
+        if (positionType == PositionOffHaveSalaryType.COMPANY_PAY) {
+            mItemHaveSalaryFlag = false;
+            notifyPropertyChanged(BR.itemHaveSalaryFlag);
+        }
+        if (positionType == PositionOffHaveSalaryType.INSURANCE_COVERAGE) {
+            mItemHaveSalaryFlag = true;
+            notifyPropertyChanged(BR.itemHaveSalaryFlag);
+        }
+    }
+
+    @Bindable
+    public String getCurrentOffType() {
+        return mCurrentOffType;
+    }
+
+    public void setCurrentOffType(String currentOffType) {
+        mCurrentOffType = currentOffType;
+        notifyPropertyChanged(BR.currentOffType);
+    }
+
+    public void setCurrentPositionOffType(int currentPositionOffType) {
+        mCurrentPositionOffType = currentPositionOffType;
+    }
+
+    @Bindable
+    public String getCurrentOffHaveSalaryType() {
+        return mCurrentOffHaveSalaryType;
+    }
+
+    public void setCurrentOffHaveSalaryType(String currentOffHaveSalaryType) {
+        mCurrentOffHaveSalaryType = currentOffHaveSalaryType;
+        notifyPropertyChanged(BR.currentOffHaveSalaryType);
+    }
+
+    public void setCurrentPositionOffHaveSalaryType(int currentPositionOffHaveSalaryType) {
+        mCurrentPositionOffHaveSalaryType = currentPositionOffHaveSalaryType;
+    }
+
+    @IntDef({
+            PositionOffType.OFF_HAVE_SALARY, PositionOffType.OFF_NO_SALARY
+    })
+    @interface PositionOffType {
+        int OFF_HAVE_SALARY = 0;
+        int OFF_NO_SALARY = 1;
+    }
+
+    @IntDef({
+            PositionOffHaveSalaryType.COMPANY_PAY, PositionOffHaveSalaryType.INSURANCE_COVERAGE
+    })
+    @interface PositionOffHaveSalaryType {
+        int COMPANY_PAY = 0;
+        int INSURANCE_COVERAGE = 1;
```

Comment:
ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.